### ISSUE TITLE
Again and again ... PostgreSQ password authentication failed 

### ISSUE THREAD

**Issue by terraboss** [goauthentik/authentik#9403]
**Describe the bug**
After updating Authentik according to the [official instructions](https://docs.goauthentik.io/docs/releases/2024.4) with the new Docker Compose file, Authentik failed to start. The log showed an error message in an endless loop stating: "INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989032.3258598 "

**To Reproduce**

1. Go to the directory containing the Docker Compose file.
2. Run `docker-compose up -d` to start Authentik.
3. Check the logs using `docker-compose logs` to observe the error.

**Expected behavior**
Authentik should start successfully without any authentication errors.

**Screenshots**
N/A

**Logs**
```
Output of docker-compose logs:
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989036.356591 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989037.374496 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989038.3930142 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989039.4110782 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989040.418761 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989041.4267035 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989042.4346516 
INF | event=PostgreSQL connection failed, retrying... (connection failed: FATAL:  password authentication failed for user "authentik") logger=authentik.lib.config timestamp=1713989043.4422038 
```

**Version and Deployment (please complete the following information):**

- Authentik version: 2024.4
- Deployment: docker-compose

**Additional context**
No additional context at this moment.

**Comment by PythonCoderAS** [comment]
Try logging in to your postgres database manually via the CLI.

**Comment by drfinkus** [comment]
@terraboss were you using Portainer to deploy, by any chance?

For anyone else reading this, I encountered the exact same issue OP did while attempting to deploy the official Docker Compose file through Portainer's Stack GUI, using the built-in environment variables editor. 

I fixed it by manually creating the .env file and using `docker compose up -d` to deploy. It no longer failed PGSQL password authentication.
